cmd/cli/flavor.go

```diff
@@ -31,10 +31,7 @@ func flavorPluginCommand(plugins func() discovery.Plugins) *cobra.Command {
 				return err
 			}
 
-			flavorPlugin, err = flavor_plugin.NewClient(endpoint.Protocol, endpoint.Address)
-			if err != nil {
-				return err
-			}
+			flavorPlugin = flavor_plugin.NewClient(endpoint.Address)
```

Comment:
Why is the protocol being removed?   We need to support named pipes per #285 and this change here will have to be undone.